reactor-core/src/withMicrometerTest/java/reactor/core/publisher/FluxTapTest.java

```diff
@@ -661,6 +712,47 @@ void doFirst() {
 				);
 		}
 
+		@Test
+		void throwingAlterContext() {
```

Comment:
It's part of a `@Nested` test. I believe the original suite was meant to group by operator type (combinations of conditonal/fuseable/regular with flux/mono) and in a nested class have all that's appropriate at the cost of duplication. Do I understand correctly that you'd like to change the paradigm - have just doFirst, throwingAlterContext, throwingCreateListener, etc. that accept consumers and make their assertions as generic validations, but have the logic in the Stream of parameters that consume a TestSubscriber? If it can be done, perhaps it can be addressed in a different PR? Rewriting it here would blur the point this PR is making. Unless of course, I misunderstood your intention?